keycloak/standalone.xml

```diff
@@ -494,6 +494,15 @@
                 <cacheTemplates>false</cacheTemplates>
                 <dir>${jboss.home.dir}/themes</dir>
             </theme>
+            <spi name="eventsListener">
+              <provider name="jboss-logging" enabled="true">
+                <properties>
+                  <!-- Overrides DEBUG default, allowing ALL login events to get logged -->
+                  <property name="success-level" value="INFO"/>
```

Comment:
It looks like this would log all account as well as login events. Is that true?

Reply:
Yes, this will actually log ALL event types, not just _login_ events.  I tweaked the comment a bit to make it more broad.